com.sigil_ebook.Sigil.appdata.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally this file should be submitted and maintained upstream.

Reply:
@dougmassay and @kevinhendricks what are your thoughts on submitting this [AppStream](https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html) metadata file (or some modified version) into upstream Sigil?